src/test/database/test_level2db.py

```diff
@@ -134,3 +155,58 @@ def test_count_failed_scans_with_limit(self, level2db):
     @pytest.mark.slow
     def test_count_failed_scans_with_offset(self, level2db):
         assert level2db.count_failed_scans(FREQMODE, offset=1) == 1
+
+
+class TestGetMeasurements:
+    @pytest.mark.parametrize("min_scanid,limit,expect", (
+        (7014791071, 50, 36),
+        (7014791071, 5, 5),
+        (7014791072, 20, 0),
```

Comment:
if that scan has exactly 20 then expect must be 20, also if such case is missing, add that case.

Reply:
these tests are enough for me.

Reply:
With current implementation you are missing case of `limit` set to `None`

Reply:
limit will never be None, minimum is now 1000 (when used by endpoint)

Reply:
OK since your change it cant be none now